redash/__init__.py

```diff
@@ -47,21 +49,23 @@ def create_redis_connection():
         else:
             db = 0
 
-        r = redis.StrictRedis(unix_socket_path=redis_url.path, db=db)
+        r = cls(unix_socket_path=redis_url.path, db=db)
     else:
         if redis_url.path:
             redis_db = redis_url.path[1]
         else:
             redis_db = 0
         # Redis passwords might be quoted with special characters
         redis_password = redis_url.password and urllib.unquote(redis_url.password)
-        r = redis.StrictRedis(host=redis_url.hostname, port=redis_url.port, db=redis_db, password=redis_password)
+        r = cls(host=redis_url.hostname, port=redis_url.port, db=redis_db, password=redis_password)
 
     return r
 
 
 setup_logging()
 redis_connection = create_redis_connection()
+walrus_db = create_redis_connection(cls=walrus.Database)
```

Comment:
Can we reuse the same connection?

Reply:
We sadly can't since the [`walrus.database.Database`](https://github.com/coleifer/walrus/blob/e9cd555a39fd9eb97d3301311f135a6c39df8b27/walrus/database.py#L58) class is a subclass of [`redis.Redis`](https://github.com/andymccurdy/redis-py/blob/3e3c96ffdbf44d6daba841b8b77f6dba4ec02e8a/redis/client.py#L2779) and not [`redis.StrictRedis`](https://github.com/andymccurdy/redis-py/blob/3e3c96ffdbf44d6daba841b8b77f6dba4ec02e8a/redis/client.py#L406) which was added to redis-py to fix some argument ordering differences compared to the Redis commands and other API and behavior inconsistencies.

I guess we could switch to using the redis.Redis based walrus Database class instead since the differences aren't that many and we simply could update our use of the methods with the different argument order. I don't mind either way.

Reply:
So ironically redis-py 3.0.0 was released today that removes the two different Redis and StrictRedis classes in favor of the StrictRedis class which will be the new default. More details: https://github.com/andymccurdy/redis-py#upgrading-from-redis-py-2x-to-30

Reply:
I wonder if there is a plan for walrus to support redis-py 3.0.0?

Reply:
Ah, d'oh, haden't caught up with my GitHub inbox: walrus 0.7.0 with redis-py 3.0.0 support is out for a few days now: https://github.com/coleifer/walrus/blob/master/CHANGELOG.md#070

Reply:
Okay, updated our code to work with the new redis-py version, but kombu needs an update to happen to land this: https://github.com/celery/kombu/commit/05152da53d169371b8b87718f33a6f47e97b7ba0

Reply:
Did they release a new version already?

Reply:
Sadly no, https://pypi.org/project/kombu/#history

Reply:
Ah, yeah, the [4.2.2 release](https://github.com/celery/kombu/compare/v4.2.1...v4.2.2) has it. I'll rebase the PR and add it to the requirements file.